src/components/Homepage/RuleItem/RuleItem.tsx

```diff
@@ -1,27 +1,36 @@
-import React from 'react'
-import { Col, Row } from 'react-grid-system';
+import React, { CSSProperties } from 'react'
 
 interface RuleItemSpec {
 	id: number,
 	title: string,
 	content?: string,
 }
 
+const roudNumberStyle: CSSProperties = {
```

Comment:
Should be renamed to roundNumberStyle (missing n)

Reply:
xD